src/comodo/TSIDController/TSIDController.py

```diff
@@ -303,6 +302,63 @@ def define_tasks(self, tsid_parameters: TSIDParameterTuning):
         self.root_link_task.set_kin_dyn(self.kindyn)
         self.root_link_task.initialize(param_handler=self.root_link_task_param_handler)
 
+        # feet regularization terms
+        self.left_foot_regularization_task = blf.tsid.VariableRegularizationTask()
+        self.left_foot_regularization_task_name = "left_foot_regularization_task"
+        self.left_foot_regularization_task_priority = 1
+        self.left_foot_regularization_task_weight = 1e-1 * np.ones(6)
+        self.left_foot_regularization_task_param_handler = (
+            blf.parameters_handler.StdParametersHandler()
+        )
+        self.left_foot_regularization_task_param_handler.set_parameter_string(
+            name="variable_name", value=self.variable_name_left_contact
+        )
+        self.left_foot_regularization_task_param_handler.set_parameter_int(
+            name="variable_size", value=6
+        )
+        self.left_foot_regularization_task.initialize(
+            param_handler=self.left_foot_regularization_task_param_handler
+        )
+
+        self.right_foot_regularization_task = blf.tsid.VariableRegularizationTask()
+        self.right_foot_regularization_task_name = "right_foot_regularization_task"
+        self.right_foot_regularization_task_priority = 1
+        self.right_foot_regularization_task_weight = 1e-1 * np.ones(6)
+        self.right_foot_regularization_task_param_handler = (
+            blf.parameters_handler.StdParametersHandler()
+        )
+        self.right_foot_regularization_task_param_handler.set_parameter_string(
+            name="variable_name", value=self.variable_name_right_contact
+        )
+        self.right_foot_regularization_task_param_handler.set_parameter_int(
+            name="variable_size", value=6
+        )
+        self.right_foot_regularization_task.initialize(
+            param_handler=self.right_foot_regularization_task_param_handler
+        )
+
+        ## Angular momentum task
+        self.angular_momentum_task = blf.tsid.AngularMomentumTask()
+        self.angular_momentum_task_name = "angular_momentum_task"
+        self.angular_momentum_task_priority = 1
+        self.angular_momentum_task_weight = 1e0 * np.ones(3)
+        self.angular_momentum_task_parameter_handler = (
+            blf.parameters_handler.StdParametersHandler()
+        )
+        self.angular_momentum_task_parameter_handler.set_group(
+            "CONTACT_0", contact_group_left
+        )
+        self.angular_momentum_task_parameter_handler.set_group(
+            "CONTACT_1", contact_group_right
+        )
+        self.angular_momentum_task_parameter_handler.set_parameter_float(
+            name="kp", value=10.0
```

Comment:
Could you add the gain as an attribute of `TSIDParameterTuning.py` as done for the CoM 
 
https://github.com/ami-iit/comodo/blob/8cbd5f5bc6f65a1385a646188abf999844244d30/src/comodo/TSIDController/TSIDParameterTuning.py#L6

only in the `__init__()` method for now is fine